django/utils/datastructures.py

```diff
@@ -109,7 +109,7 @@ def __deepcopy__(self, memo=None):
 
     def __getstate__(self):
         obj_dict = self.__dict__.copy()
-        obj_dict['_data'] = {k: self.getlist(k) for k in self}
+        obj_dict['_data'] = {k: self._getlist(k) for k in self}
```

Comment:
True since that only deals with existing keys. I though that it's better to return original list than make extra copy again.